nvtabular/torch_dataloader.py

```diff
@@ -125,7 +125,7 @@ def create_tensors_plain(gdf, cat_cols=None, cont_cols=None, label_cols=None):
 
 
 def combine_tensors(cats, conts, label):
-    cats_list = [cats[x] for x in _get_embedding_order(cats.keys())] if cats else None
+    cats_list = [cats[x] for x in sorted(cats.keys())] if cats else None
```

Comment:
There's one other instance of `_get_embedding_order` in this file. For clarify / consistency could consider making this change there, too

Reply:
fixed